index.php

```diff
@@ -9,10 +9,17 @@
 use NYPL\Starter\ErrorHandler;
 
 try {
-    Config::initialize(__DIR__);
+    Config::initialize(__DIR__ . '/config');
 
     $service = new Service();
 
+    $service->get("/docs/doc", function (Request $request, Response $response) {
```

Comment:
So is /docs/doc the path that gets hit when you go to platform?

Reply:
`/docs/doc` is the endpoint that serves the Swagger docs for this service itself. Essentially the documentation here: https://platformdocs.nypl.org/#/docs/getDocs.